### PR TITLE
added array based version of TestStore.assert

### PR DESCRIPTION
**Pull request by pteasima**

### CODE REVIEW

File: Sources/ComposableArchitectureTestSupport/TestStore.swift [hunk 1]
```diff
@@ -161,6 +161,15 @@ extension TestStore where LocalState: Equatable {
     _ steps: Step...,
     file: StaticString = #file,
     line: UInt = #line
+  ) {
+    assert(steps, file: file, line: line)
+  }
+  
+  /// Asserts against an array of actions.
```
Comment (pteasima) [inline Sources/ComposableArchitectureTestSupport/TestStore.swift:168]: not sure about the comment, didnt want both to have the same one 😅

**Comment by pteasima** [comment]
cmon, how to I get conflicts, I pulled. Ignore until I sort this out

File: ComposableArchitecture.xcworkspace/contents.xcworkspacedata [hunk 2]
```diff
@@ -7,9 +7,6 @@
    <FileRef
       location = "group:Examples/CaseStudies/CaseStudies.xcodeproj">
    </FileRef>
-   <FileRef
-      location = "group:Examples/LocationManager/LocationManager.xcodeproj">
-   </FileRef>
```
Comment (mbrandonw) [inline ComposableArchitecture.xcworkspace/contents.xcworkspacedata:12]: We don't wanna remove this, can we bring it back?

Comment (pteasima) [reply ComposableArchitecture.xcworkspace/contents.xcworkspacedata:12]: working on it now. Sorry about that

**Comment by pteasima** [comment]
Well, that was embarassing :D all good now?

**Comment by mbrandonw** [comment]
Looks good!

**Pull request merged**